addons/web_tour/views/tour_views.xml

```diff
@@ -1,5 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+        <record id="edit_tour_form" model="ir.ui.view">
+            <field name="model">web_tour.tour</field>
+            <field name="arch" type="xml">
+                <form string="Menu" editable="bottom">
+                    <sheet>
+                        <group>
+                            <group>
+                                <field name="name"/>
+                            </group>
+                            <group>
+                                <field name="user_id"/>
+                            </group>
+                        </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
         <record id="edit_tour_list" model="ir.ui.view">
```

Comment:
`create="0"` would probably make sense here as well, I'm not able to manually create this kind of record even as admin

Reply:
Also existing editable="bottom" seems to have no effect on this list view either.

Reply:
updated this too